pkgs/tools/networking/obfs4/default.nix

```diff
@@ -2,25 +2,28 @@
 
 buildGoModule rec {
   pname = "obfs4";
-  version = "0.0.14";
+  version = "0.1.0";
 
   src = fetchFromGitLab {
     domain = "gitlab.torproject.org";
     group = "tpo";
     owner = "anti-censorship/pluggable-transports";
-    repo = "obfs4";
-    rev = "obfs4proxy-${version}";
-    sha256 = "sha256-/d1qub/mhEzzLQFytgAlhz8ukIC9d+GPK2Hfi3NMv+M=";
+    # We don't use pname = lyrebird and we use the old obfs4 name as the first
+    # will collide with lyrebird Gtk3 program.
```

Comment:
These comments seems to be at the wrong place. Shouldn't it be just before `pname` ?

Reply:
> These comments seems to be at the wrong place. Shouldn't it be just before `pname` ?

It's just that this is the 1st place where `lyrebird` shows up, so it explains for a reader reading the expression sequentially why is this the `pname` and `rev`.